lib/signable/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module Signable
-  VERSION = "0.0.7"
+  VERSION = '0.0.8'
```

Comment:
We were discussing semantic versioning the other day.
Should we start using it now do you think?
If the third digit is for bug-fixes, and the second digit is for additions without breaking changes, wouldn't adding Rubocop be a second digit change?
So `0.1.7` ?   Or did I misunderstand?

Reply:
no in this case it would be 0.1.0.

Reply:
Shall we do that then?

Reply:
let's 489079f